packages/react-app/src/components/giveaways/Card.tsx

```diff
@@ -1,83 +1,22 @@
 import { format } from 'date-fns';
-import React, { useEffect, useMemo, useRef, useState } from 'react';
+import { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 
 import { Box, Card, CardContent, Skeleton, Typography } from '@mui/material';
 
 import Trophy from '../../assets/Trophy.png';
 import useUserInfo from '../../hooks/useUserInfo';
 import { GetParticipants } from '../../lib/queryClient';
-import { Giveaway, ParticipationState, UserRole } from '../../lib/types';
+import { Giveaway, ParticipationState } from '../../lib/types';
 import ParticipationService from '../../services/giveawayparticipation';
-import {
-  ApprovalPendingButton,
-  CheckingButton,
-  ManageButton,
-  NotAllowedButton,
-  ParticipateButton,
-  ParticipatingButton,
-} from './ActionButtons';
-import { PendingLocationModal } from './StatusModals';
+import ParticipationButton from './ParticipationButton';
 
-const ActionButtonComponents: { [K in ParticipationState]: React.FC<any> } = {
-  manage: ManageButton,
-  participating: ParticipatingButton,
-  pending: ApprovalPendingButton,
-  allowed: ParticipateButton,
-  notAllowed: NotAllowedButton,
-  checking: CheckingButton,
-  rejected: NotAllowedButton,
-};
-
-type GiveawayCardProps = {
-  giveaway: Giveaway;
-  onParticipationError: () => void;
-  onRejection?: () => Promise<void>;
-};
-
-const GiveawayCard = ({
-  giveaway,
-  onParticipationError,
-  onRejection,
-}: GiveawayCardProps) => {
+const GiveawayCard = (giveaway: Giveaway) => {
   const navigate = useNavigate();
   const userInfo = useUserInfo();
-  const { data: participants, refetch } = GetParticipants(giveaway._id);
-  const [showPendingModal, setShowPendingModal] = useState(false);
-  const participationAction = useRef(false);
-  const isAdmin = userInfo?.role === UserRole.ADMIN;
+  const { data: participants } = GetParticipants(giveaway._id);
```

Comment:
```suggestion
  const { data: participants } = getParticipants(giveaway._id);
```

Reply:
This must be pascal case since it creates an abstraction with useQuery hook

Reply:
In case case it should be called `useGetParticipants` since this is an hook. Pastel case is for components